scripts/start

```diff
@@ -9,7 +9,7 @@ projectRoot="$(a="/$0"; a=${a%/*}; a=${a:-.}; a=${a#/}/; cd "$a/.." || return; p
 
 cd "${projectRoot}"
 
-docker-compose down -v
+docker-compose down
```

Comment:
Do not remove named volumes. The means files added to the Azurite container are persisted between restarts.